roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -8,7 +8,7 @@ metadata:
     deployment: "{{deploy_name}}"
     logging-infra: "{{logging_component}}"
 spec:
-  replicas: {{replicas|default(1)}}
+  replicas: {{es_replicas|default(1)}}
```

Comment:
... when would you even have a case where you wanted something other than 1 ES replica?